File: app-api/src/main/java/be/insaneprogramming/cleanarch/boundary/EvictTenantFromBuilding.java
package be.insaneprogramming.cleanarch.boundary;

import be.insaneprogramming.cleanarch.requestmodel.EvictTenantFromBuildingRequest;

@FunctionalInterface
@Boundary
public interface EvictTenantFromBuilding {
	void execute(EvictTenantFromBuildingRequest request);
}

File: app-api/src/main/kotlin/be/insaneprogramming/cleanarch/boundary/EvictTenantFromBuilding.kt
package be.insaneprogramming.cleanarch.boundary

import be.insaneprogramming.cleanarch.requestmodel.EvictTenantFromBuildingRequest

@FunctionalInterface
interface EvictTenantFromBuilding {
    fun execute(request: EvictTenantFromBuildingRequest)
}
